Classes/Service/Tika/TikaServiceFactory.php

```diff
@@ -37,7 +37,7 @@ class TikaServiceFactory
     public static function getTika($tikaService = '')
     {
         if (empty($tikaService)) {
-            $settings = unserialize($GLOBALS['TYPO3_CONF_VARS']['EXT']['extConf']['extractor']);
+            $settings = $GLOBALS['TYPO3_CONF_VARS']['EXTENSIONS']['extractor'];
```

Comment:
ExtensionConfiguration

Reply:
I'm sure that this will work, but there is a new Class in TYPO3 called ExtensionConfiguration->get($extKey). So, there is an official API. I would prefer to use it.

Reply:
done